templates/typescript-app/.hooks.sscaff.js

```diff
@@ -18,7 +18,7 @@ exports.post = ctx => {
       '@types/jest@26',
       'jest@26',
       'ts-jest@26',
-      'typescript'
```

Comment:
This is to fix the failing integ test, which currently fails with:

```console
binary npm was executed with.

> cdk8s-init-test-OjsHef@1.0.0 compile /tmp/cdk8s-init-test-OjsHef
> tsc --build

Error: tsconfig.json(4,5): error TS5101: Option 'charset' is deprecated and will stop functioning in TypeScript 5.5. Specify compilerOption '"ignoreDeprecations": "5.0"' to silence this error.
```

TypeScript 5.0 was [released](https://github.com/microsoft/TypeScript/releases/tag/v5.0.2) 3 days ago, and they introduced a few [deprecations](https://devblogs.microsoft.com/typescript/announcing-typescript-5-0/#deprecations-and-default-changes). 

I'm not sure why these deprecations turn into errors that fail `cdk8s import`, but we shouldn't have been picking up a new ts major version anyway.